content/kapp-controller/docs/latest/packaging.md

```diff
@@ -119,7 +119,7 @@ spec:
   licenses:
   - "Apache 2.0"
   - "MIT"
-  # Timestamp of release (iso8601 formatted string; optional)
+  # Timestamp of release (optional; metav1.Time (recommended to use iso8601 formatted string))
```

Comment:
lets leave this as iso8601. metav1.Time is a pretty internal implementation detail.

Reply:
Yeah, makes sense. I can close this then.